symengine/symengine_exception.h

```diff
@@ -1,12 +1,14 @@
 #ifndef SYMENGINE_EXCEPTION_H
 #define SYMENGINE_EXCEPTION_H
 
-enum symengine_exceptions_t {
-    NO_EXCEPTION,
+typedef enum {
+    NO_EXCEPTION = 0,
     DIV_BY_ZERO,
     NOT_IMPLEMENTED,
     UNDEFINED
```

Comment:
Since this is C where there are no namespaces, it is better to use `SYMENGINE_` prefix.